### ISSUE TITLE
[Feature] markLine in polar

### ISSUE THREAD

**Issue by hllshiro** [apache/echarts#20317]
### Version

5.5.1

### Link to Minimal Reproduction

https://echarts.apache.org/examples/zh/editor.html?c=line-polar&code=MYewdgzgLgBAJgQygmBeGBtAugbgFABmIATjABQA2AprAJZowAMOM9APOgIyPOsDUfAJQwA3nhgxqsKAAsaKdGXoB6GN0bCAVDADMANmbjJNGKXQBWGNrKcYfGAFkkMgHQRaYMmVnyYqzgAcGlaOzi4ACgCSgoL4EojILgAOAK4QMmTEsXgAvnggSVC04AxiEkVQ1ABcokblVAAeUDUA5AAqAO4gMABqCBQpVAC0AIINVBCsYDDhIBQIxC1GOQA0RtQA5lRgcDVlEvBICDUYLRQeVC1Yy2sSSXMLe6tGUCBzRUl7dTBQxLQbW2IrQQDVoEBatwOILBsw8UCoQNqBwOUAAnkkqK1gMQQBBwd88hJnhIEGANtQxmCvsi0RjWgA3fqDCHfaALKAjMnVJg3IzEBBwWhpSkQJ6QiAI2gTE7ffbI0AkQVgJBUADKqOgVAAtq17vNFpDkcqtZiYC0IMAkPCDd9yujTebLVBrSzkfEjjUEghDQctQsANYAGQu1LdEnc1DAzRgBH6Ep9yK9obDEkYyZTEmNDognFdGYOCuIu0wjBW5iwCZT5zAarR3Ll-YkoAoJFaAGICAQeDw842JB1aHBZDUAEyl235idhwn54n5zjpjNZ1oQEe9_OF4sYfSl8uV5Ez6cTw8wGfXHI4IA

### Steps to Reproduce

1. open the link
2. run code to preview

### Current Behavior

Testing polar's markLine function in the official sample program does not show the expected effect

### Expected Behavior

A circle should be drawn at the position 5

### Environment

```markdown
- OS:windows 11
- Browser: edge 128.0.2739.54
- Framework: none
```

### Any additional comments?

[Official Documents](https://echarts.apache.org/zh/option.html#series-scatter.markLine.data.0.type) does not mention that MarkLine cannot be used in polar mode, and the description of the coord property specifically mentions polar mode. But the sample program didn't work as expected

**Comment by Ovilia** [comment]
Yes, markLine is not supported in polar. You may use `graphic.elements` to draw the shape you need.

**Comment by hllshiro** [comment]
Thanks, I'll try `graphic.elements` later.
